tools/internal_ci/macos/grpc_objc_bazel_test.sh

```diff
@@ -100,7 +100,7 @@ objc_bazel_tests/bazel_wrapper \
   $BAZEL_FLAGS \
   --test_env HOST_PORT_LOCAL=localhost:$PLAIN_PORT \
   --test_env HOST_PORT_LOCALSSL=localhost:$TLS_PORT \
-  --test_env FLAKE_TEST_REPEATS=3 \
+  --test_env FLAKE_TEST_REPEATS=10 \
```

Comment:
10 retries for a test seems crazy high. Why would you ever need so many retries for a test? If you really need that many retries it means that the test itself is probably garbage and adds almost no value.
We should focus on how to make individual invocations of the test more reliable.

Reply:
Let's revert this line.